jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
-  testMatch: ["<rootDir>/test/system/**/*.spec.ts"],
+  testMatch: ["<rootDir>/test/**/*.spec.ts"],
```

Comment:
This makes also the `windows` tests run unfortunately! There might be a way to exclude the windows path here as a new entry, something like `"!<rootDir>/test/windows/**/*"`

Reply:
Yeah, exactly, I'm in progress of fixing it 😄 🙏